lib/fsext/filepath.go

```diff
@@ -0,0 +1,17 @@
+// Package fsext provides extended file system functions
+package fsext
+
+import (
+	"path"
+	"path/filepath"
+)
+
+// JoinFilePath is a wrapper around filepath.Join
+// starting go 1.20 on Windows, Clean does not modify the
+// volume name other than to replace occurrences of "/" with `\`.
+// that's why we need to add a leading slash to the path
+// go.1.19: filepath.Join("\\c:", "test")  // \c:\test
+// go.1.20: filepath.Join("\\c:", "test")  // \c:test
```

Comment:
@olegbespalov I think in this comment, we're missing why we're using `path` instead of `filepath`.

Reply:
@codebien sorry, I'm missing the point :cry: The comment is about the behavior change behavior of the Clean, which is used underneath the `filepath.Join`

But for sure, feel free to suggest a comment that you think will be more informative.

Reply:
I mean why `path.Clean` and not `filepath.Clean`

Reply:
`path.Clean` and `filepath.Clean` are pretty similar in their behavior, and the thing is that in that particular case, it doesn't matter which one we use. The outcome remains. 

We kind of force to treat the second argument of  `filepath.Join` as the one that "has slash" regardless it has it or not. And later filepath.Join applies cross-platform manipulations.

If you feel that the `filepath.Join` is more appropriate here, I could change it :+1:

Reply:
>The filepath package uses **either forward slashes or backslashes**, depending on the operating system. To process paths such as URLs that **always use forward slashes** regardless of the operating system, see the path package. 

We are using both, so my interpretation from the docs is that `filepath` could be more consistent with it.

Reply:
yeah, but as I said we use the `filepath.Join` later, but okay I'll change it for the sake of the consistency